cocos/renderer/gfx-vulkan/VKCommands.cpp

```diff
@@ -744,9 +744,11 @@ void cmdFuncCCVKCreateRenderPass(CCVKDevice *device, CCVKGPURenderPass *gpuRende
 }
 
 void cmdFuncCCVKCreateFramebuffer(CCVKDevice *device, CCVKGPUFramebuffer *gpuFramebuffer) {
-    size_t              colorViewCount = gpuFramebuffer->gpuColorViews.size();
-    bool                hasDepth       = gpuFramebuffer->gpuRenderPass->depthStencilAttachment.format != Format::UNKNOWN;
-    vector<VkImageView> attachments(colorViewCount + hasDepth);
+    size_t                  colorViewCount = gpuFramebuffer->gpuColorViews.size();
+    bool                    hasDepth       = gpuFramebuffer->gpuRenderPass->depthStencilAttachment.format != Format::UNKNOWN;
+    vector<VkImageView>     attachments(colorViewCount + hasDepth);
+    VkFramebufferCreateInfo createInfo{VK_STRUCTURE_TYPE_FRAMEBUFFER_CREATE_INFO};
+    createInfo.width = createInfo.height = UINT_MAX;
```

Comment:
Better check attachments total count > 0 to ensure the function is safe by itself